recipe-server/parquet_schemas/normandy.admin.docker.app/request.summary.cfg

```diff
@@ -0,0 +1,37 @@
+-- vim:ft=lua
+message_matcher = message_matcher .. " && Fields[Type] == 'request.summary'"
+
+s3_path_dimensions = {
+	{name="log", source="Type"},
+	{name="type", source="Fields[Type]"},
+	{name="date", source="Fields[Date]"},
+	{name="hour", source="Fields[Hour]"},
+}
+
+parquet_schema = [=[
+message Log {
+    required int64 Timestamp;
+    optional binary Type (UTF8);
+    optional binary Hostname (UTF8);
+    optional binary Logger (UTF8);
+    optional binary EnvVersion (UTF8);
+    optional int64 Severity;
+    optional int64 Pid;
+    optional binary fields_lang (UTF8);
+    optional binary fields_method (UTF8);
+    optional binary fields_path (UTF8);
+    optional binary fields_rid (UTF8);
+    optional binary fields_uid (UTF8);
+    optional binary fields_user_agent_browser (UTF8);
+    optional binary fields_user_agent_os (UTF8);
+    optional binary fields_user_agent_version (UTF8);
```

Comment:
you won't always get the raw agent, because that's PII, but for agents that fail to parse I can keep the agent.

mozlog json is converted to heka/hindsight messages here:
https://github.com/mozilla-services/cloudops-deployment/blob/parquet_rework/libs/puppet/modules/logging/files/hindsight/decoders/mozlog.lua

and then fields are removed/transformed here:
https://github.com/mozilla-services/cloudops-deployment/blob/parquet_rework/libs/puppet/modules/logging/files/hindsight/encoders/common.lua

the default config options are the ones in place unless configured in these files.